omdb/src/bin/omdb/main.rs

```diff
@@ -35,8 +36,9 @@ async fn main() -> Result<(), anyhow::Error> {
         .context("failed to create logger")?;
 
     match args.command {
-        OmdbCommands::Nexus(nexus) => nexus.run_cmd(&log).await,
         OmdbCommands::Db(db) => db.run_cmd(&log).await,
+        OmdbCommands::Nexus(nexus) => nexus.run_cmd(&log).await,
+        OmdbCommands::Sled(sled) => sled.run_cmd(&log).await,
```

Comment:
I only wonder if this should be `sled-agent` (there are other things on the sled).  If you do I'd carry that through everywhere -- e.g., the URL env var name too.

Reply:
Done!